.eslintrc.js

```diff
@@ -4,7 +4,8 @@ module.exports = {
     parser: '@typescript-eslint/parser',
     jsx: true,
     useJSXTextNode: true,
-    allowImportExportEverywhere: true
+    allowImportExportEverywhere: true,
+    sourceType: "module"
```

Comment:
What does this do?

Reply:
My Webstorm was complaining about `ESLint: Parsing error: 'import' and 'export' may appear only with 'sourceType: module'`. So this is the way to tell eslint we allow ECMAScript modules.

Reply:
Okay